### ISSUE TITLE
Update to newer Nextcloud version

### ISSUE THREAD

**Issue by NikoQT** [nextcloud/backup#720]
The stable channel only provides this plugin for Nextcloud 28.x

thats really sad :(

I really would like to update my nextcloud instance to new versions

**Comment by smarinier** [comment]
See #663, the stable28 doesn't work in NC 29, see corresponding PR #725

**Comment by NikoQT** [comment]
> See [#663](https://github.com/nextcloud/backup/issues/663), the stable28 doesn't work in NC 29, see corresponding PR [#725](https://github.com/nextcloud/backup/pull/725)

I know. My questions was when the update releases

**Comment by Bubba8291** [comment]
@smarinier how do I use your PR without running into integrity check issues?

I am having issues with a SFTP remote point, and I think it might be related to the integrity check failing.

**Comment by smarinier** [comment]
@Bubba8291 I don't really know how it works with your (i suppose) Nextcloud support licence, but if there are Integrity Checks, you are not supposed to change filles. I think you should check with the Nextcloud support directly.

**Comment by sternj** [comment]
I'm basically seeing no activity from contributors on this repo in... a long time, aside from CI modifications. The most recent activity I'm seeing that's non-trivial was back in January, and it also seemed to be (largely) updates

**Comment by smarinier** [comment]
Hi @ArtificialOwl, would it be possible to have a look on PR #725, unblock this and build a release  ? Backup works mainly as a cron, and this makes silent errors, and no backup... Thanks

**Comment by Tampa** [comment]
How is an app that writes in big letters about keeping data safe and secure neglecting the very basis of that by not updating a backup app. Heck, how is this not a built-in function in the first place. 3-2-1 backups are the norm, the minimum, these days. The PR looks simple enough, it shouldn't take that much to test and approve it.

**Comment by sbe-arg** [comment]
I moved personal nextcloud instances to nextcloud-snap just for the simplicity of snap backups.

Business and enterprise grade setups probably get better assurance using other OS wide-r level backup solutions.

Was disappointing to see a silent drop of support of this app.

**Comment by Tampa** [comment]
Now that the pr is merged when can we expect an updated release to come out? The app hasn't seen a new version in a year according to the app store!

**Comment by DaanSelen** [comment]
Will this be updated to work with version 31?

**Comment by niels-beier** [comment]
Is there any update on this topic?

**Comment by Tampa** [comment]
Nextcloud wants to be the new office alternative. Well guess what, that requires solid backup tools. It's insane this is being completely ignored. The most basic IT 101, having backups, should get a bit more priority, especially since it's now mostly fixed and just needs a release. Or is there something still broken?

@artonge What's stopping a release?

**Comment by Tampa** [comment]
IT HAS BEEN A YEAR. For crying out loud. You can't claim Nextcloud being "safe home for your data" without a built-in backup solution.

**Comment by NikoQT** [comment]
> IT HAS BEEN A YEAR. For crying out loud. You can't claim Nextcloud being "safe home for your data" without a built-in backup solution.

I dont think this is developed anymore. In case you didn't knew, this was an unoffical plugin. 
I guess you are supposed to use a raid system for nextcloud. 
Now I just use a cron job to backup the database

**Comment by Tampa** [comment]
Raid is not a backup and there is a lot of stuff involved in making backups manually. When the plugin does all that already it seems simpler to just continue working on that instead. It just doesn't make sense that Nextcloud themselves market it as a safe haven for data when there isn't even a decent backup solution built into it. Every other major system has this, from Plesk to Wordpress to Proxmox. Backups are one of the most important parts of any software that handles sensitive data, so this exclusion is like shipping a bowl of ramen without broth.
